content/intro-to-storybook/react/en/get-started.md

```diff
@@ -73,4 +73,14 @@ svn export https://github.com/chromaui/learnstorybook-code/branches/master/publi
 svn export https://github.com/chromaui/learnstorybook-code/branches/master/public/font public/font
 ```
 
+<div class="aside">
+  <p>Additionally, if you prefer <code>curl</code> and <code>tar</code> to download a folder of files from GitHub instead use the below command.</p></div>
+  
+```bash
+curl https://codeload.github.com/chromaui/learnstorybook-code/tar.gz/master --output c.gz 
+tar -C public -xvzf c.gz --strip=2 learnstorybook-code-master/public/font 
+tar -C public -xvzf c.gz --strip=2 learnstorybook-code-master/public/icon 
+rm c.gz
+```
```

Comment:
@aaroncalderon @domyen  and @tmeasday. I was checking and it looks like we can do something like:

```bash
curl https://codeload.github.com/chromaui/learnstorybook-code/tar.gz/master --output assets.gz

tar -C public -xvzf c.gz --strip=2 learnstorybook-code-master/public/font | tar -C public -xvzf c.gz --strip=2 learnstorybook-code-master/public/icon

rm assets.gz
```

Text wise we could change it to:

```html
<div class="aside">
<p>We’ve used curl and tar to easily get the most recent assets from GitHub. If you don’t have curl installed or want to just do it manually, you can grab the folders directly here (<- the here link stays the same).</p>
<p>Depending on your OS, you probably need to update some of the commands to allow the assets to be copied without any issues.</p>
</div>
```


Probably something like this? 

And we could drop the svn alltogether.

Feel free to provide feedback

Reply:
I think this oneliner works: 

```
curl https://codeload.github.com/chromaui/learnstorybook-code/tar.gz/master  | tar -zxvC public --strip=2 learnstorybook-code-master/public/{font,icon}
```

WDYT @jonniebigodes?

Reply:
@tmeasday let me check it out and get back to you.

Reply:
@tmeasday did a run with it and what you posted works fine on a WSL, meaning probably all the NIX based should be good to go. So i think that with the adjustment to this command we can streamline and remove svn in here. Also if the reader is running Windows we've covered that mentioning that he needs to adjust the command.

Reply:
Ok, sounds good to me @jonniebigodes

Reply:
@aaroncalderon when you're able, can you address the change and commit it, so that we can this merged?

Reply:
got it...

Working on it.

Reply:
> curl https://codeload.github.com/chromaui/learnstorybook-code/tar.gz/master  | tar -zxvC public --strip=2 learnstorybook-code-master/public/{font,icon}

@tmeasday I had issues with piping the output of `curl` to `tar`. That is why I split the command so I would get a file, then i would use the file to extract the desired files and finally remove the temporary file.

I would like to keep the command in separate lines. Although that one liner looks neat.